### ISSUE TITLE
Android emulator changes?

### ISSUE THREAD

**Issue by pors** [meteor/meteor#5358]
Following the instructions here https://github.com/meteor/meteor/wiki/Mobile-Development-Install:-Android-on-Mac no longer seems to work.

```bash
$ meteor --version
Meteor 1.2.0.2

$ meteor run android
[[[[[ ~/dev/tora ]]]]]

=> Started proxy.
=> Started MongoDB.
=> Started your app.

=> App running at: http://localhost:3000/
=> Errors executing Cordova commands:

   While running Cordova app for platform Android with options --emulator:
   Error: Command failed: /Users/mark/dev/tora/.meteor/local/cordova-build/platforms/android/cordova/run --emulator
   /Users/mark/dev/tora/.meteor/local/cordova-build/platforms/android/cordova/node_modules/q/q.js:126
   throw e;
   ^
   ERROR : No emulator images (avds) found.
   1. Download desired System Image by running: /Users/mark/Library/Android/sdk/tools/android sdk
   2. Create an AVD by running: /Users/mark/Library/Android/sdk/tools/android avd
   HINT: For a faster emulator, use an Intel System Image and install the HAXM device driver
   at ChildProcess.exitCallback (/tools/utils/processes.js:137:23)
   at ChildProcess.emit (events.js:98:17)
   at Process.ChildProcess._handle.onexit (child_process.js:820:12)

ExitWithCode:1
```

Is this a newer version than used in the wiki?

<img width="336" alt="screen shot 2015-10-03 at 12 08 45" src="https://cloud.githubusercontent.com/assets/1078320/10262237/8c06c314-69c7-11e5-8e40-b428f702eb08.png">

There is a difference with the screenshots in the wiki, there is another window now when clicking "SDK Manager".

What I tried:

* I installed all System Images I thought might be relevant (everything except Wear and TV).
* I created an AVD by running /Users/mark/Library/Android/sdk/tools/android avd
* This resulted in actually starting an emulator, but never loading the meteor app.

**Comment by martijnwalraven** [comment]
There may have been changes in Android Studio 1.4, which was only released a few days ago and I haven't been able to test yet. But I believe it should still create an AVD for you. With the current version of Cordova, you'll have to make sure you have Android SDK Platform 22 installed.

You mention the emulator starts up but the Meteor app doesn't load. Do you mean `meteor run android` does start the emulator? So you no longer get the error message listed above? Any other errors?

**Comment by pors** [comment]
Yes I installed Platform 22, but it didn't create an AVD.
I created one myself and ```meteor run android``` does start the emulator, but it keeps on hanging.
This is the output:

```
Preparing Cordova project for platform Android
Running Cordova app for platform Android with options --emulator
ANDROID_HOME=/Users/mark/Library/Android/sdk |
JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk1.8.0_60.jdk/Contents/Home
=> Started your app.

=> App running at: http://localhost:3000/
WARNING : no emulator specified, defaulting to TestAVD
Waiting for emulator...
Creating filesystem with parameters:         /
    Size: 576716800
    Block size: 4096
    Blocks per group: 32768
    Inodes per group: 7040
    Inode size: 256
    Journal blocks: 2200
    Label:
    Blocks: 140800
    Block groups: 5
    Reserved block group size: 39
Created filesystem with 11/35200 inodes and 4536/140800 blocks
ERROR: couldn't get path to resize2fs binary \
Creating filesystem with parameters:
    Size: 69206016
    Block size: 4096
    Blocks per group: 32768
    Inodes per group: 4224
    Inode size: 256
    Journal blocks: 1024
    Label:
    Blocks: 16896
    Block groups: 1
    Reserved block group size: 7
Created filesystem with 11/4224 inodes and 1302/16896 blocks
emulator: The memory needed by this AVD exceeds the max specified in your HAXM configuration.
emulator: AVD      RAM size = 3072 MB
emulator: HAXM max RAM size = 2048 MB
emulator: You might want to adjust your AVD RAM size and/or HAXM configuration to run in fast virt mode.

HAXM is not working and emulator runs in emulation mode
emulator: emulator window was out of view and was recentered

emulator: UpdateChecker: skipped version check
     Starting app on Android Emulator     
```

**Comment by martijnwalraven** [comment]
It seems something may have gone wrong with the configuration of the AVD. The procedure might differ between versions, but you should be able to follow the [Android Studio documentation](http://developer.android.com/tools/devices/managing-avds.html). Make sure to select API 22 as the system version for the device.

**Comment by pors** [comment]
I tried this, and I got it to work. Earlier attempts with the GUI AVD tool didn't work.

```
$ android list targets
Available Android targets:
----------
id: 1 or "android-22"
     Name: Android 5.1.1
     Type: Platform
     API level: 22
     Revision: 2
     Skins: HVGA, QVGA, WQVGA400, WQVGA432, WSVGA, WVGA800 (default), WVGA854, WXGA720, WXGA800, WXGA800-7in
 Tag/ABIs : default/armeabi-v7a, default/x86, default/x86_64

$ android create avd -n myAndroid -t 1 --abi default/x86_64
Android 5.1.1 is a basic Android platform.
Do you wish to create a custom hardware profile [no]
Created AVD 'myAndroid' based on Android 5.1.1, Intel Atom (x86_64) processor,
with the following hardware config:
hw.lcd.density=240
hw.ramSize=512
vm.heapSize=48
```

During the first run a lot of *.pom files were downloaded BTW.

**Comment by zapstar** [comment]
+1 @pors . BTW @martijnwalraven  I made it work with the GUI AVD by first launching the app on linux console.. I get stuck where @pors mentioned... then I pressed <kdb>Ctrl</kdb>+<kdb>C</kdb> to kill the  process on linux, and finally re-run the command in the command line.

**Comment by JernejJerin** [comment]
I did have the same issue as @pors about missing emulator image which I downloaded when setting up AVD through AVD manager. The [help guide actually](https://github.com/meteor/meteor/wiki/Mobile-Development-Install:-Android-on-Linux) says to deselect all, even the **Intel x86 Atom_64 System Image**, when setting up in Android SDK Manager. Which I do not quite understand as if you want to set up AVD, you will need one of this images.
